grpcmock-spring-boot/src/test/java/org/grpcmock/springboot/TestBase.java

```diff
@@ -23,13 +24,21 @@ abstract class TestBase {
 
   @Value("${grpcmock.server.port}")
   protected int grpcMockPort;
+
+  @Value("${grpcmock.server.name}")
+  private String inProcessName;
+
   private ManagedChannel serverChannel;
 
   @BeforeEach
   void setupChannel() {
-    serverChannel = ManagedChannelBuilder.forAddress("localhost", grpcMockPort)
-        .usePlaintext()
-        .build();
+    if (grpcMockPort == -1) {
```

Comment:
Could detect it, based on whether the `inProcessName` is defined or not.

Reply:
Indeed, it's better approach. Fixed.